src/cleanlab_codex/utils/aws.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List
+
+from pydantic import BaseModel
+
+from cleanlab_codex.utils.types import FunctionParameters
+
+
+class ToolSpec(BaseModel):
+    name: str
+    description: str
+    inputSchema: Dict[str, FunctionParameters]  # noqa: N815
```

Comment:
```suggestion
    input_schema: Dict[str, FunctionParameters] = Field(alias="inputSchema")
```

Not sure what coding style @axl1313 wants to follow in this repo, but you could use [Pydantic Aliases](https://docs.pydantic.dev/latest/concepts/alias/) to avoid the camelcase Python field.

Reply:
Thanks for the suggestion!